content/zh/blog/2023/experimental-dual-stack/index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "支持双栈 Kubernetes 集群"
+description: "对双栈 Kubernetes 集群的实验性支持。"
+publishdate: 2023-03-10
+attribution: "张怀龙 (Intel), 徐贺杰 (Intel), 丁少君 (Intel), Jacob Delgado (F5), 蔡迎春 (前 F5)"
+keywords: [双栈]
+---
+
+在过去的一年里，英特尔和 F5 在为 Istio 提供 [Kubernetes 双栈网络](https://kubernetes.io/docs/concepts/services-networking/dual-stack/)的支持中通力合作。
+
+## 背景
```

Comment:
Please add an anchor for the header.

Reply:
Hi @rootsongjc, what do you mean the anchor for the header?

Reply:
Hi @rootsongjc, okay, done, thanks!